app/assets/stylesheets/styles.scss

```diff
@@ -0,0 +1,31 @@
+h1,
+h2,
+h3 {
+  font-weight: lighter;
+}
+
+h1 a {
+  color: #222222;
+}
+
+h1 a:hover {
+  color: #43AC6A;
```

Comment:
Color literals like `#43AC6A` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#43AC6A` should be written as `#43ac6a`